examples/simple-html-consumer/static/index.html

```diff
@@ -28,6 +28,7 @@
         units: 'metric',
       }
       var myMap = farmOS.map.create("map", options);
+      myMap.addBehavior("image");
```

Comment:
I would second this but otherwise, I think it looks amazing @paul121. Maybe a download button first would be more intuitive? Or just the standard three dots if that is an option? I like that it is a drop down though to reduce the clutter on the screen.

Reply:
Agree on "Snapshot"! I spoke with @aislinnpearson and we agreed as implemented is a good start for now.